src/bin/server.rs

```diff
@@ -8,15 +8,21 @@
 
 use mini_redis::{server, DEFAULT_PORT};
 
+#[cfg(feature = "xray")]
```

Comment:
Could all the X-ray feature gated imports be grouped with a comment explaining what they are for

Reply:
ok